lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -681,26 +681,31 @@ module.exports = function(self, options) {
   };
 
   self.addDocReferencesMigration = function() {
-    self.apos.migrations.add(self.__meta.name + '.docReferences', function(callback) {
+
+    // This migration is needed because formerly,
+    // docs that only referenced this attachment via
+    // a join were counted as "owning" it, which is
+    // incorrect and leads to failure to make it
+    // unavailable at the proper time
+
+    self.apos.migrations.add(self.__meta.name + '.docReferencesCorrected', function(callback) {
```

Comment:
I suggest `docReferencesContained` or the like so as to not tie naming to whether it was or wasn't at one point working.